mbf_abstract_nav/include/mbf_abstract_nav/abstract_recovery_execution.h

```diff
@@ -41,14 +41,22 @@
 #ifndef MBF_ABSTRACT_NAV__ABSTRACT_RECOVERY_EXECUTION_H_
 #define MBF_ABSTRACT_NAV__ABSTRACT_RECOVERY_EXECUTION_H_
 
-#include <pluginlib/class_loader.h>
+#include <map>
+#include <string>
+#include <stdint.h>
+#include <vector>
+
 #include <boost/chrono/thread_clock.hpp>
+#include <boost/shared_ptr.hpp>
+#include <boost/thread.hpp>
+
+#include <pluginlib/class_loader.h>
```

Comment:
same same: There is no class loader anymore in mbf_abstract_nav

Reply:
agree